### ISSUE TITLE
[Discuss] need to add the daemonstatus api for the new version of scrapyd 

### ISSUE THREAD

**Issue by BruceDone** [djm/python-scrapyd-api#7]
Hi all:
 since the scrapyd server has add the new api of daemonstatus.json 

 here is the code 
https://github.com/scrapy/scrapyd/blob/master/scrapyd/webservice.py

here is the doc 
https://scrapyd.readthedocs.org/en/latest/api.html#daemonstatus-json

i think may be need to update the code and add this new api to the project

**Comment by Digenis** [comment]
daemonstatus.json is not in a scrapyd release yet
but I don't see why it shouldn't make it to the next.

**Comment by djm** [comment]
Yup, as @Digenis noted in #5, this is not in a stable branch yet.

It has also changed within the last month so I'm not certain we should be relying on it being a stable API just yet.

> but I don't see why it shouldn't make it to the next.

@Digenis do you have any more information on this? I can't find much info set in stone about scrapyd's release process.

**Comment by Digenis** [comment]
The change was only a bugfix.
I don't think we should drop daemonstatus from scrapyd
There are other recently introduced features that should be dropped.
The reason you can't rely on it is because it's not tagged in a release yet.

Regarding releases, there are currently no conventions.
The odd-even convention for unstable-stable releases
is finally abandoned by scrapy.
I'm currently doing an effort to fix the setups of existing releases of scrapyd
which have erroneously remained with obsolete scrapy dependencies
and python2.6 support for too long. I think the setup scripts are important
because of the amount of people who accidentally end up with broken installations.
I'm done with the 1.0 branch, I'll do the same on 1.1
and then some cleanup and testing towards a 1.2 release.
Any help on this is welcome.

**Comment by djm** [comment]
Thanks @Digenis.

I will likely start work on supporting `daemonstatus` in a branch our side so that we are ready to release as soon as `scrapyd` is at the point where it can be. 

I'll leave this ticket open to track progress.

**Comment by tyong920** [comment]
daemonstatus.json is now stable. Please add it.
